packages/calypso-e2e/src/lib/blocks/block-flows/all-form-fields.ts

```diff
@@ -52,32 +59,43 @@ export class AllFormFieldsFlow implements BlockFlow {
 			[ 'Date Picker', 'Add label…' ],
 			[ 'Phone Number Field', 'Add label…' ],
 			[ 'Multi-line Text Field', 'Add label…' ],
-			[ 'Checkbox', 'Add label…' ],
+			[ 'Checkbox', isRefactor ? 'Add option…' : 'Add label…' ],
```

Comment:
This one might be a bug in my branch. 'Add label' seems like it'd be slightly better for a checkbox.

I'll look into it.

Reply:
I've changed it in this commit - https://github.com/Automattic/jetpack/pull/43767/commits/efd48821b2626660579cf6732537f6d3771c07ea.

It should be possible to change this line back now.

```suggestion
			[ 'Checkbox', 'Add label…' ],
```